earn-with-merkl/earning-with-merkl.md

```diff
@@ -78,12 +78,20 @@ You can claim all your rewards per chain at once to optimize gas costs!
 
 Rewards on Merkl do not increase block by block, but can be claimed at a frequency which depends on the chain. You can check the claim frequency on the [Status page](https://app.merkl.xyz/status).
 
-Note that, by default, rewards can only be claimed by the address that earned them. You can however approve an operator to claim on your behalf by calling the function `toggleOperator` on the [distributor smart contract](https://app.merkl.xyz/status). However, rewards will still be sent to the original address that earned them.
+### Operator System: Delegating Claim Rights
 
-So to sum up, assuming Alice earned the rewards:
+By default, rewards can only be claimed by the address that earned them. However, Merkl provides a flexible operator system that allows you to delegate the right to claim rewards to another address, while still receiving the rewards yourself.
 
-* by default only Alice can claim and rewards are sent to Alice.
-* by calling `toggleOperator`, Alice can allow Bob to claim on her behalf. Then, Bob can claim for Alice by sending Alice's proof to the contract, and rewards are then sent to Alice.
+**How regular operators work:**
+
+You can approve an operator to claim on your behalf by calling the function `toggleOperator` on the [distributor smart contract](https://app.merkl.xyz/status). When an operator claims on your behalf, the rewards are still sent to your original address—the operator only facilitates the transaction.
```

Comment:
can you add in this section that if you toggleOperator address(0) then anyone can claim on your behalf